src/app/searchResults/searchResults.tpl.html

```diff
@@ -129,14 +129,14 @@ <h4 class="is-results-divider" ng-if="facet === 'featured' && $ctrl.featuredGrou
                     </a>
                     <div class="is-row-meta">
                       <a ng-href="{{$ctrl.buildVanity(r.path)}}">
-                        <span class="is-row-title">{{r.name}}</span>
+                        <span id="result-{{r.designationNumber}}" class="is-row-title">{{r.name}}</span>
```

Comment:
There is a chance this will produce duplicates

Reply:
Good point, I'll link them by id instead of designation number.